app/templates/org/view_profile.html

```diff
@@ -132,7 +132,7 @@ <h3>Gallery</h3>
   </div>
 </section>
 <section style="padding-top: 10vh">
-          {% include 'partials/_footer.html'%}
+          {% include 'partials/_secondary_footer.html'%}
```

Comment:
what's the difference between these 2 footers?

Reply:
yea its based off the mockups so footer has the header Ready to connect while secondary_footer does not.

Reply:
oh i see. let's leave the regular footer in and we'll make changes later if needed 😀